ftw/book/layer.py

```diff
@@ -11,11 +12,29 @@ class BookTraverse(DefaultPublishTraverse):
 
     def publishTraverse(self, request, name):
 
-        layout_layer_name = getattr(self.context, 'latex_layout', '')
-        if layout_layer_name:
-            layout_layer = resolve(layout_layer_name)
-            provide_request_layer(request, layout_layer)
-
-        provide_request_layer(request, IWithinBookLayer)
+        provideBookLayers(self.context, request)
 
         return super(BookTraverse, self).publishTraverse(request, name)
+
+
+def provideBookLayers(context, request):
```

Comment:
I'd prefer lowercase here -> `provide_book_layers`.
I'd also rename `context` to `book` since it always has to be a book.